core/server/guilds.js

```diff
@@ -79,27 +102,35 @@ Meteor.methods({
     });
     analytics.updateGuild(Guilds.findOne(guildId), {}, Meteor.userId());
 
+    emitMemberEvent(guildId);
     log('addGuildUsers: done');
   },
-  removeTeamUser(guildId, userId) {
-    check([guildId, userId], [Match.Id]);
-    log('removeTeamUser: start', { guildId, userId });
+  removeTeamUsers(guildId, userIds) {
+    check(guildId, Match.Id);
+    check(userIds, [Match.Id]);
+    log('removeTeamUsers: start', { guildId, userIds });
 
+    if (!userIds.length) return;
     if (!this.userId) throw new Meteor.Error('not-authorized', 'User not allowed');
 
-    if (!canEditGuild(Meteor.user(), Guilds.findOne(guildId))) throw new Meteor.Error('not-authorized', `Missing permissions to edit team members`);
+    const guild = Guilds.findOne(guildId);
+    if (!canEditGuild(Meteor.user(), guild)) throw new Meteor.Error('not-authorized', `Missing permissions to edit team members`);
 
-    const user = Meteor.users.findOne(userId);
-    if (user.guildId !== guildId) throw new Meteor.Error('user-invalid', 'Given user is not in the team');
+    let users = Meteor.users.find({ _id: { $in: userIds }, guildId });
+    if (users.length === 0) throw new Meteor.Error('user-invalid', 'Given user is not in the team');
```

Comment:
This part is wrong, in fact in the previous code it was checked that the deleted user was in the team.But I wonder if this check is really necessary 🤔

Reply:
I think it's important if selected user is in another team we have to throw an error, but I have to change ```(users.length === 0)``` to ```(users.length === userIds.length)```